### ISSUE TITLE
no matches found: /home/chengjianhua/.antigen-etc/bundle.d/*.zsh

### ISSUE THREAD

**Issue by chengjianhua** [gantsign/ansible_role_antigen#50]
After performing ansible playbook which configures the role like below:
```
    - role: gantsign.antigen
      users:
        - username: "{{ ansible_user }}"
          antigen_libraries:
            - name: oh-my-zsh
```

Then I login into my remote host, I encountered the following errors:
```
/home/chengjianhua/.antigenrc:14: no matches found: /home/chengjianhua/.antigen-etc/bundle.d/*.zsh
```

It seems that the empty bundle list caused glob expression to fail:
https://github.com/gantsign/ansible_role_antigen/blob/2e3f5719e852de66458eeb4f9ec67e2385173303/templates/antigenrc.j2#L12

**Comment by chengjianhua** [comment]
@freemanjp Could you take this a look, please 🙏

**Comment by freemanjp** [comment]
Hi, @chengjianhua thanks for reporting this issue and sorry for the late reply (my day job hasn't left much time for my OSS projects recently). I never anticipated anyone would use the role with the `antigen_libraries` without also specifying some `antigen_bundles`. Specifying at least one bundle e.g. `git` should work around the issue for now. I doubt I'll have time for a proper fix until next year.

**Comment by freemanjp** [comment]
@chengjianhua sorry it took some time, the fix is in version [1.4.2](https://github.com/gantsign/ansible_role_antigen/releases/tag/1.4.2) of this role.

**Comment by chengjianhua** [comment]
> @chengjianhua sorry it took some time, the fix is in version [1.4.2](https://github.com/gantsign/ansible_role_antigen/releases/tag/1.4.2) of this role.

@freemanjp Appreciate your great work 👍
